src/handle_action.cpp

```diff
@@ -966,22 +966,29 @@ static void sleep()
     }
 
     time_duration try_sleep_dur = 24_hours;
+    std::string deaf_text = "";
+    if( g->u.is_deaf() ) {
+        deaf_text = "<color_c_red> (DEAF!)</color>";
```

Comment:
You need to translate this.
```suggestion
        deaf_text = _( "<color_c_red> (DEAF!)</color>" );
```

Reply:
Do I just commit that or is there more I need to do to make it translatable?

Reply:
[See the docs](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/TRANSLATING.md) for more info, but this string will be available for translation if you do this, yes.